src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -5266,10 +5266,14 @@ void MacroAssembler::char_array_compress(Register src, Register dst, Register le
 // by the call to JavaThread::aarch64_get_thread_helper() or, indeed,
 // the call setup code.
 //
-// aarch64_get_thread_helper() clobbers only r0, r1, and flags.
+// On Linux, aarch64_get_thread_helper() clobbers only r0, r1, and flags.
+// On Windows, the helper is a usual C function.
```

Comment:
This should say "other systems", not "Windows". Otherwise OK.

Reply:
Thanks, fixed.